services/common/src/components/projectSummary/DocumentUpload.tsx

```diff
@@ -63,15 +43,17 @@ export const DocumentUpload: FC<DocumentUploadProps> = (props) => {
   };
 
   useEffect(() => {
-    props.change(FORM.ADD_EDIT_PROJECT_SUMMARY, "spatial_documents", []);
-    props.change(FORM.ADD_EDIT_PROJECT_SUMMARY, "support_documents", []);
+    dispatch(change(FORM.ADD_EDIT_PROJECT_SUMMARY, "spatial_documents", []));
```

Comment:
Thanks for setting up the documents table to show the uploaded files without needing to go back to see them